test/integration/admin/api/member_permissions_controller_test.rb

```diff
@@ -151,7 +199,7 @@ class Admin::Api::MemberPermissionsControllerTest < ActionDispatch::IntegrationT
     put admin_api_permissions_path(id: @user.id, format: :json), { allowed_sections: ['[]'] }
 
     @user.member_permissions.reload
-    assert_empty @user.allowed_sections
+    assert_empty @user.allowed_sections.to_a
```

Comment:
Why? 🤔 Without the conversion to Array, it is a Set, so empty should be empty anyway 🤔

Reply:
It was a "replace all" `@user.allowed_sections` to `@user.allowed_sections.to_a` 😅